.github/workflows/make-live.yml

```diff
@@ -0,0 +1,80 @@
+name: Make Live
+# Make live versions of .Rmd files in training modules
+
+# This workflow only runs when it is manually dispatched
+on:
+  workflow_dispatch:
+    inputs:
+      skiprendering:
+        description: Should rendering be skipped? TRUE or FALSE
+        required: true
+        default: 'FALSE'
+
+jobs:
+  # This workflow contains a single job called "build-all"
+  make-live:
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+    container:
+      image: ccdl/training_dev:latest
+
+    steps:
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+      - name: Checkout code
+        uses: actions/checkout@v2
+        with:
+          # get the full repo
+          fetch-depth: 0
+          # use alexslemonade-docs-bot
+          token: ${{ secrets.DOCS_BOT_GITHUB_TOKEN }}
+
+      # Download data
+      #- name: Download data
+      #  run: bash scripts/download-data.sh
+      #- name: Render all pages to html
+      #  run: |
+      #    docker run \
+      #    --mount type=bind,target=/home/rstudio,source=$PWD \
+      #    ccdl/training_dev:latest \
+      #    snakemake --cores 2 --forceall
+
+      # Run the make-live.R script but skip rendering
+      - name: Run make-live.R with skiprendering
+        if: skiprendering == 'TRUE'
+        id: make_live skiprendering
+        run: Rscript --vanilla scripts/make-live.R --skiprendering
+
+      # Run the make-live.R script but do NOT skip rendering
+      - name: Run make-live.R with rendering
+        if: skiprendering == 'FALSE'
+        id: make_live
+        run: Rscript --vanilla scripts/make-live.R
```

Comment:
I'm not sure this approach where we _repeat_ a step in a workflow with different logic is more desirable than moving from  using `--skiprendering` which stores `TRUE` to something like `--rendering` which is passed `TRUE` or `FALSE` in `make-live.R`.

I'm also not sure this works in its current state. I would expect this to require you to use `${{ github.event.inputs.skiprendering }}` in the `if`. (This seems difficult to test.)

Reply:
> --skiprendering which stores TRUE to something like --rendering which is passed TRUE or FALSE in make-live.R.

Ah this is not what I understood from when we talked about this "in person". I thought you were referring to having `--skiprendering` take a TRUE/FALSE as an option which felt less intuitive. So just make `--skiprendering` into `--rendering` but how does that allow us to not repeat a step? 

> I'm also not sure this works in its current state. I would expect this to require you to use ${{ github.event.inputs.skiprendering }} in the if. (This seems difficult to test.)

Yeah I wasn't sure how to test this before merging. But I think you are right about the `${{ github.event.inputs.skiprendering }}`.

Reply:
> but how does that allow us to not repeat a step?

```
        run: Rscript --vanilla scripts/make-live.R --rendering ${{ github.event.inputs.skiprendering }}
```

You may need quotes around that, though.

Reply:
Oh so we do want to make it so you have to say `TRUE/FALSE` for rendering.

Reply:
Yes, that's what I meant by:

> `--rendering` which is passed `TRUE` or `FALSE` in `make-live.R`.
>